src/engine-actions/index.js

```diff
@@ -65,7 +64,6 @@ module.exports = {
     isBalanceSufficient,
     payInvoice,
     prepareSwap,
-    executeSwap,
```

Comment:
So is `settleSwap` replacing `executeSwap` now? I'd be interested in learning the story behind this.

Reply:
`initiateSwap` replaced `executeSwap`